NanoAnalysis/python/modules/eleScaleResProducer.py

```diff
@@ -32,7 +32,7 @@ def getEleScaleRes(era, tag, is_mc, overwritePt=True, EtDependent=None):
                     smearKey = "2022Re-recoE+PromptFG_SmearingJSON" if is_mc else None
                     fname = "electronSS_Standard_2022postEE.json.gz"
 
-    elif era == 2023:
+    elif era == 2023 or era == 2024 or era == 2025:
```

Comment:
As this is a hack for 24 and 25, can we add a warning message - similar to what done [here](https://github.com/CJLST/ZZAnalysis/blob/Run3/AnalysisStep/src/LeptonSFHelper.cc#L76) - so that we don't forget?